include/Options/Business/Domains.php

```diff
@@ -185,4 +185,21 @@ protected function sanitize( $value, Options $options ) {
 	protected function get_description(): string {
 		return __( 'Domains used when the language is set from different domains.', 'polylang' );
 	}
+
+	/**
+	 * Adds information to the site health info array.
+	 *
+	 * @since 3.8
+	 *
+	 * @param array   $info    The current site health information.
+	 * @param Options $options An instance of the Options class providing additional configuration.
+	 *
+	 * @return array The updated site health information.
+	 */
+	public function add_to_site_health_info( array $info, Options $options ): array { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		if ( 3 === $options->get( 'force_lang' ) ) {
+			$info = $this->get_site_health_info( $info, $this->get(), self::key() );
+		}
+		return $info;
```

Comment:
```suggestion
		if ( 3 === $options->get( 'force_lang' ) ) {
			return $this->get_site_health_info( $info, $this->get(), self::key() );
		}
		
		return $info;
```
No reassignment needed.